cloudinit/cmd/query.py

```diff
@@ -65,6 +65,21 @@ def get_parser(parser=None):
     return parser
 
 
+def maybe_decode_decompress_userdata(ud_file_path):
```

Comment:
Minor nit: "maybe" functions always bother me. Could it be something like `load_userdata`, or `obtain_userdata`?

If you like it how it is though, no need to change it.

Reply:
really good point, I just went with other naming conventions of some functions throughout cloud-init, but I tend to agree with you. A function should probably not be named based on what steps it may or may not perform, but what it gives you.
renamed `load_userdata` which also saves us a bunch of characters and a couple of userless newlines for formatting :)